### DISCUSSION TITLE
Test progress logger format

### DISCUSSION THREAD

**Discussion by marioaag** [avocado-framework/avocado#5352] [Q&A] [answered]
Hi @richtja,

Following the question done in #5304, Do you know if is a way to get rid of the pre-appended "avocado.test.progress" string to the message?
Example of what I'm getting in the terminal:
```
JOB ID     : fc9ae67862f706dcde1de3021279c6131cd716ce
JOB LOG    : /tmp/avocado_logs/job-2022-04-25T23.05-fc9ae67/job.log
 (220425-210559294061.CMD.-1/1) /home/mario/git/Gatekeepers/test/avocado/testcases/ErrorInjects/ErrorInjects.py:ErrorInjects.node_remove_replace: STARTED
220425-210559294061.CMD.-1-/home/mario/git/Gatekeepers/test/avocado/testcases/ErrorInjects/ErrorInjects.py:ErrorInjects.node_remove_replace: avocado.test.progress: Running Inject 0

RESULTS    : PASS 0 | ERROR 0 | FAIL 0 | SKIP 1 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /tmp/avocado_logs/job-2022-04-25T23.05-fc9ae67/results.html
JOB TIME   : 17.34 s
2022-04-25 23:06:16 -I- BlueGen: ***Avoc
```

In the documentation doesn't show avocado.test.progress string in the messages, the difference here is that I'm doing the configuration by my self using the API but don't think that it should make a difference in the result.

I'm sending the message as simple as: 
```
prog_log.info(f'Running Inject {inject_number}')
```

**Comment by richtja** [comment, marked as answer]
Hi @marioaag this is desired behavior, the whole pre-appended message `220425-210559294061.CMD.-1-/home/mario/git/Gatekeepers/test/avocado/testcases/ErrorInjects/ErrorInjects.py:ErrorInjects.node_remove_replace:` is for debugging purposes and IMO is even necessary when you will use parallel run with multiple tests and multiple streams. Can you please show me where the documentation doesn't show the `avocado.test.progress` string, because I found it in [documentaion](https://avocado-framework.readthedocs.io/en/latest/guides/writer/chapters/logging.html?#showing-custom-log-streams)? Definitely this should be in documentation and if we missing it we need to fix it.

**Reply by marioaag** [reply]
Hi @richtja, thanks for the confirmation, I found that I was looking at the 92.0 documentation and as you pointed in the latest it is there!
